AFQ/api/group.py

```diff
@@ -146,6 +161,8 @@ def __init__(self,
         if not isinstance(bids_layout_kwargs, dict):
             raise TypeError("bids_layout_kwargs must be a dict")
 
+        parallel_params["engine"] = parallel_params.get("engine", "serial")
```

Comment:
I am getting some strange behaviors on hyak and wondering whether this is causing some issues, so I am going to remove it for now:
```suggestion
```